packages/features/bookings/Booker/Booker.tsx

```diff
@@ -0,0 +1,196 @@
+import type { MotionStyle } from "framer-motion";
+import { LazyMotion, domAnimation, m, AnimatePresence } from "framer-motion";
+import { Fragment, useEffect } from "react";
+import StickyBox from "react-sticky-box";
+import { shallow } from "zustand/shallow";
+
```

Comment:
This file is the main entrypoint for the booker. This loads all different sections.